src/core/public/core_app/styles/_mixins.scss

```diff
@@ -112,3 +112,25 @@
     }
   }
 }
+
+@mixin kbnThemeStyle($theme, $mode: 'both') {
+  @if ($mode == 'both') {
+    $themeLight: $theme + 'light';
```

Comment:
If I'm understanding, this is appending 'light', correct?
Should the $kbnThemeVersion values (in each of the 4 globals files) just be `v7` or `v8`?

Reply:
Added some validations to make it clearer what the mixin accepts as $theme ('v7' and 'v8').